jstreegrid.js

```diff
@@ -216,7 +218,7 @@
 					gs.gridcontextmenu = false;
 				}
 				// find which column our tree shuld go in
-				for (i=0;i<s.columns.length;i++) {
+        for (var i = 0, len = s.columns.length;i<len;i++) {
```

Comment:
Hey - it was about profiling. So caching length is faster. ;-)
But yes - there is not much of a speed gain.